brianlong/www.validators.app
# 184822722 - fix sol price component

#### What's this PR do?
Set proper sol price refresh delay. Utilize database price in case of CoinBase response is incorrect.

#### How should this be manually tested?
Test the following:
- Coinbase values (should be refreshed each 30 seconds by the daemon) after the sol price component is mounted
- try to create situation when the Coinbase error raises. During that the db price should be fetched and presented if no Coinbase values has been stored before.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184822722)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
